README.md

```diff
@@ -352,61 +353,74 @@ This command will start the container and map port 8000 of the container to port
 
-To run the docker image from `Dockerfile` locally: 
-
-```bash
-
-docker run -v ~/.aws:/root/.aws -d -p 8000:8000 borehole-api
-```
-
-To run the Docker image from `Dockerfile` with the environment variables from the `.env` file
+To run the Docker image from `Dockerfile` with the environment variables from the `.env` file:
 
 ```bash
 docker run --env-file .env -d -p 8000:8000 borehole-api
 ```
 
-To run the docker image used for AWS Lambda: `Dockerfile.aws.lambda`: 
-
-```bash 
-docker run --platform linux/amd64 -v ~/.aws:/root/.aws -d -p 8000:8000 borehole-api:test
-```
-
 4.2.2. **Passing the AWS credentials as Environment Variables**
 
-It is also possible to set the AWS credentials as you environment variables and the environment variables of the Docker image you are trying to run. 
+It is possible to set AWS credentials as environment variables and also pass them to the Docker image you are running.
 
-Unix-based Systems (Linux/macOS)
+**Unix-based Systems (Linux/macOS)**
 
 Add the following lines to your `~/.bashrc`, `~/.bash_profile`, or `~/.zshrc` (depending on your shell):
 
 ```bash
 export AWS_ACCESS_KEY_ID=your_access_key_id
 export AWS_SECRET_ACCESS_KEY=your_secret_access_key
 export AWS_ENDPOINT=your_endpoint_url
+export AWS_S3_BUCKET=your_bucket_name
 ```
 
-Please note that the endpoint url is in the following format: `https://{bucket}.s3.<RegionName>.amazonaws.com`. This 
-URL can be found in AWS when you go to your target S3 bucket, select any item in the bucket and look into the 
-Properties under `Object URL`. Please remove the file specific extension and you will end up with your endpoint URL. 
+Notes:
+
+- The data extraction API in this repository is designed to be integrated into [swissgeol-boreholes-suite](https://github.com/swisstopo/swissgeol-boreholes-suite) that is configure by [swissgeol-boreholes-config](https://github.com/swisstopo/swissgeol-boreholes-config). [Here](https://github.com/swisstopo/swissgeol-boreholes-config/blob/ac293abe1c489044b3b15efa30c2238d456ded26/charts/swissgeol-boreholes/values.yaml#L65) you can find more information about the AWS S3 Bucket Configuration and the environment variables defined for it. 
+- The `AWS_ENDPOINT` follows the format: `https://s3.<RegionName>.amazonaws.com`. You can find this URL in AWS by navigating to your target S3 bucket, selecting any item, and checking its Properties under "Object URL." Remove the file-specific extension to obtain the correct endpoint URL.
+- In some environments, the `AWS_ENDPOINT` may look like this:
+    
+    a. **Kubernetes cluster environment**
+
+    ```bash
+    AWS_ENDPOINT=https://s3.<region>.amazonaws.com
+    AWS_S3_BUCKET=your-[dev|prod]-bucket
+    ```
+
+    b. **In local boreholes development**, a local S3-compatible service like [MinIO](https://min.io/) is used, configured as:
```

Comment:
@stijnvermeeren-swisstopo I added an indent here because it was a bit clearer. I remember that you spotted some issues with thepast indents in the . Could you check if it work on your end? Otherwise I will remove them.